pages/History.qml

```diff
@@ -596,6 +597,19 @@ Rectangle {
                         radius: 8
                         color: isout ? "#d85a00" : "#2eb358"
                     }
+
+                    MoneroComponents.TextPlain {
+                        visible: isFailed || isPending
+                        anchors.top: parent.top
+                        anchors.topMargin: 24
+                        anchors.horizontalCenter: parent.horizontalCenter
+                        font.family: FontAwesome.fontFamilySolid
+                        font.styleName: isFailed ? "Solid" : ""
+                        font.pixelSize: 15
+                        text: isFailed ? FontAwesome.times : FontAwesome.clockO
+                        color: isFailed ? "#FF0000" : MoneroComponents.Style.defaultFontColor
```

Comment:
Please post a screenshot of `Pending` state

Reply:
![image](https://user-images.githubusercontent.com/45968869/101152096-df6a3680-3622-11eb-8a5e-cb214f4247de.png)

Reply:
Shouldn't it be green (it's still an incoming tx)?

Reply:
I prefer displaying the clock icon of pending transactions in white, because their status is riskier than confirmed transactions.

Only safe transactions (with at least one confirmation) are displayed in color: orange for sent and green for received txs.